{"version":"3.8.2","messages":["\nAn error occurred while loading ./spec/features/users/sign_in_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0mfeature \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mSign in\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m, \u001b[33m:devise\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m# Scenario: User cannot sign in if not registered\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Given I do not exist as a user\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   When I sign in with valid credentials\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Then I see an invalid credentials message\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0mscenario \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31muser cannot sign in if not registered\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m      \u001b[0msignin(\u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mtest@example.com\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m, \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mplease123\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m)\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mexpect\u001b[0m(page).to have_content(\u001b[35m\u001b[1;35m/\u001b[0m\u001b[35m#{I18n.t(\u001b[1;35m'\u001b[0m\u001b[35mdevise.failure.not_found_in_database\u001b[1;35m'\u001b[0m\u001b[35m\u001b[0m\u001b[35m, authentication_keys: \u001b[1;35m'\u001b[0m\u001b[35memail\u001b[1;35m'\u001b[0m\u001b[35m\u001b[0m\u001b[35m)}\u001b[0m\u001b[35m\u001b[1;35m/\u001b[0m\u001b[35m\u001b[35mi\u001b[0m\u001b[35m\u001b[0m)\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNoMethodError:\u001b[0m\n\u001b[31m  undefined method `feature' for main:Object\u001b[0m\n\u001b[36m# ./spec/features/users/sign_in_spec.rb:7:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `block in load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n","\nAn error occurred while loading ./spec/features/users/sign_out_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0mfeature \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mSign out\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m, \u001b[33m:devise\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m# Scenario: User signs out successfully\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Given I am signed in\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   When I sign out\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Then I see a signed out message\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0mscenario \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31muser signs out successfully\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0muser = \u001b[1;34;4mFactoryGirl\u001b[0m.create(\u001b[33m:user\u001b[0m)\u001b[0m\n\u001b[31m      \u001b[0msignin(user.email, user.password)\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mexpect\u001b[0m(page).to have_content \u001b[1;34;4mI18n\u001b[0m.t \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mdevise.sessions.signed_in\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNoMethodError:\u001b[0m\n\u001b[31m  undefined method `feature' for main:Object\u001b[0m\n\u001b[36m# ./spec/features/users/sign_out_spec.rb:5:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `block in load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n","\nAn error occurred while loading ./spec/features/visitors/about_page_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0mfeature \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mAbout page\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m# Scenario: Visit the 'about' page\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Given I am a visitor\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   When I visit the 'about' page\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Then I see \"About the Website\"\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0mscenario \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mVisit the about page\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0mvisit \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31m/about\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mexpect\u001b[0m(page).to have_content \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mAbout the Website\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNoMethodError:\u001b[0m\n\u001b[31m  undefined method `feature' for main:Object\u001b[0m\n\u001b[36m# ./spec/features/visitors/about_page_spec.rb:5:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `block in load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n","\nAn error occurred while loading ./spec/features/visitors/home_page_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0mfeature \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mHome page\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m# Scenario: Visit the home page\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Given I am a visitor\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   When I visit the home page\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Then I see \"Welcome\"\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0mscenario \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mvisit the home page\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0mvisit root_path\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mexpect\u001b[0m(page).to have_content \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mWelcome\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNoMethodError:\u001b[0m\n\u001b[31m  undefined method `feature' for main:Object\u001b[0m\n\u001b[36m# ./spec/features/visitors/home_page_spec.rb:5:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `block in load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n","\nAn error occurred while loading ./spec/features/visitors/navigation_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0mfeature \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mNavigation links\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m, \u001b[33m:devise\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m# Scenario: View navigation links\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Given I am a visitor\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   When I visit the home page\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Then I see \"home,\" \"sign in,\" and \"sign up\"\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0mscenario \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mview navigation links\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0mvisit root_path\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mexpect\u001b[0m(page).to have_content \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mHome\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mexpect\u001b[0m(page).to have_content \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mSign in\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNoMethodError:\u001b[0m\n\u001b[31m  undefined method `feature' for main:Object\u001b[0m\n\u001b[36m# ./spec/features/visitors/navigation_spec.rb:5:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `block in load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n","\nAn error occurred while loading ./spec/features/visitors/sign_up_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0mfeature \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mSign Up\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m, \u001b[33m:devise\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m# Scenario: Visitor can sign up with valid email address and password\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Given I am not signed in\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   When I sign up with a valid email address and password\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[1;30m#   Then I see a successful sign up message\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0mscenario \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mvisitor can sign up with valid email address and password\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0msign_up_with(\u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mtest@example.com\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m, \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mplease123\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m, \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mplease123\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m)\u001b[0m\n\u001b[31m      \u001b[0mtxts = [\u001b[1;34;4mI18n\u001b[0m.t( \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mdevise.registrations.signed_up\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m), \u001b[1;34;4mI18n\u001b[0m.t( \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mdevise.registrations.signed_up_but_unconfirmed\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m)]\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mexpect\u001b[0m(page).to have_content(\u001b[35m\u001b[1;35m/\u001b[0m\u001b[35m.*#{txts[0]}\u001b[0m\u001b[35m.*|.*#{txts[1]}\u001b[0m\u001b[35m.*\u001b[1;35m/\u001b[0m\u001b[35m\u001b[0m)\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNoMethodError:\u001b[0m\n\u001b[31m  undefined method `feature' for main:Object\u001b[0m\n\u001b[36m# ./spec/features/visitors/sign_up_spec.rb:5:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `block in load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n","\nAn error occurred while loading ./spec/models/user_spec.rb.\n\u001b[31mFailure/Error: \u001b[0mrequire \u001b[1;34;4mFile\u001b[0m.expand_path(\u001b[31m\u001b[1;31m'\u001b[0m\u001b[31m../../config/environment\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m, \u001b[1;36m__FILE__\u001b[0m)\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mFrozenError:\u001b[0m\n\u001b[31m  can't modify frozen Array\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/engine.rb:577:in `unshift'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/engine.rb:577:in `block in \u003cclass:Engine\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:30:in `instance_exec'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:30:in `run'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:55:in `block in run_initializers'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:44:in `each'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:44:in `tsort_each_child'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:44:in `each'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:44:in `tsort_each_child'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:44:in `each'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:44:in `tsort_each_child'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:44:in `each'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:44:in `tsort_each_child'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/initializable.rb:54:in `run_initializers'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/railties-4.2.11.1/lib/rails/application.rb:352:in `initialize!'\u001b[0m\n\u001b[36m# ./config/environment.rb:5:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:274:in `require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:274:in `block in require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:274:in `require'\u001b[0m\n\u001b[36m# ./spec/rails_helper.rb:3:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:274:in `require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:274:in `block in require'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:274:in `require'\u001b[0m\n\u001b[36m# ./spec/models/user_spec.rb:1:in `\u003ctop (required)\u003e'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `block in load'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:240:in `load_dependency'\u001b[0m\n\u001b[36m# /usr/local/rvm/gems/ruby-2.6.1/gems/activesupport-4.2.11.1/lib/active_support/dependencies.rb:268:in `load'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.000453216,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":8},"summary_line":"0 examples, 0 failures, 8 errors occurred outside of examples"}